config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
+  resources :users
   resources :cards
   root "pages#welcome"
   get "random" => "cards#random"
   put "check" => "cards#check"
   patch "check" => "cards#check"
+  get 'login' => 'user_sessions#new', :as => :login
+  post 'logout' => 'user_sessions#destroy', :as => :logout
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.